src/app/shared/form-field/form-field.component.html

```diff
@@ -1,20 +1,48 @@
 <div class="row">
   <div class="col-sm-4">
-    <label>{{ field[0].label }}:</label>
-      
-    <div *ngIf="field[0].type == 'select'">   
-      <select [ngFormControl]="field[1]"   
+    <label>{{ field[0].label }}</label>
+
+    <div *ngIf="field[0].type == 'select'">
+      <select #select  [ngFormControl]="field[1]"
+        (change)="updateValue($event)"
```

Comment:
instead of updating the "FormManger" collection at every input change, we can now define when we want this to happen. Like in text fields its on blur (it could also be a custom event).